recipe/meta.yaml

```diff
@@ -1,48 +1,53 @@
 {% set name = "httpcore" %}
-{% set version = "0.15.0" %}
+{% set version = "1.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 18b68ab86a3ccf3e7dc0f43598eaddcf472b602aba29f9aa6ab85fe2ada3980b
+  sha256: 9fc092e4799b26174648e54b74ed5f683132a464e95643b226e00c2ed2fa6535
+  patches:
+    - patches/0001-remove-fancy-pypi-readme.patch
```

Comment:
Is this because the required files aren't in the pypi package?

Reply:
https://github.com/AnacondaRecipes/hatch-fancy-pypi-readme-feedstock/pull/2#pullrequestreview-1852098078